src/launcher/containers/ReleaseNotesDialogContainer.js

```diff
@@ -55,6 +55,7 @@ function mapStateToProps({
             releaseNote: app.releaseNote,
             canUpdate: app.currentVersion !== app.latestVersion,
             latestVersion: app.latestVersion,
+            isInstalled: app.currentVersion != null,
```

Comment:
I am slightly unhappy about this line: That a defined `currentVersion` implies that the app is installed might seem obvious but implications like these is not something a container should do. And looking around, I saw in other places code which concluded from a defined `path` that the app is installed.

It would be better, if this knowledge is bundled in a place that just knows these things about apps. But currently I do not see that we have such a place. [`apps.js`](https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/blob/30680d0f36378fa3a07e9ee952cb31d490361a5f/src/main/apps.js) is where most knowledge about apps is held, but that file is more about retrieving handling app files (which is also why it is usually executed in the main thread). [`models/index.js`](https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/blob/30680d0f36378fa3a07e9ee952cb31d490361a5f/src/launcher/models/index.js) is also about apps, but more about putting them in an immutable wrapper.

So, for now I grumpily just leave this code here, until we find or create a better place for it.